### DISCUSSION TITLE
part-ii/2017-86/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#397] [Question comments]
Part II 2017 Logic and Set Theory: Paper 3, Section II, H 

https://questions.tripos.org/part-ii/2017-86/

**Comment by artie2000** [comment]
All but last part is bookwork. AoC is used to find mappings of elements to greater elements and chains to upper bounds.

<details>
<summary>Last part:</summary>

Let $\mathcal{B}$ be a basis for $V$ over $\mathbb{Q}$. If $\mathcal{B}$ is finite, then we are done by IB LinAlg. 

If $\mathcal{B}$ is infinite, let $\kappa=\text{card}\mathcal{B}$. Mapping finite linear combinations to their coefficients, have a bijection

$V\leftrightarrow\bigsqcup_{j=0}^\infty\mathcal{B}^j\mathbb{Q}^j$

Taking cardinals of both sides and applying cardinal arithmentic rules, get

$\text{card}\mathcal{B}^j\mathbb{Q}^j=\kappa^j\aleph_0^j=\kappa$

$\text{card}V=\aleph_0\kappa=\kappa$.

So all bases have the same cardinality as $V$.

</details>
